tests/functions.sh

```diff
@@ -52,6 +59,29 @@ function ssh_ping {
     fi
 }
 
+function ssh_install {
+    id=$1
+    #installing netcat
+    ssh_run $id pacman -S --noconfirm netcat &> /dev/null
+}
+
+function udp_test {
+    id1=$1
+    id2=$2
+    ssh_install $1
+    ssh_install $2
+    ssh_run_background $id1 "nc -lup 5554 > /tmp/test"
```

Comment:
I think you can use -w to kill netcat, and I don't think -u is useful in listening mode.